src/components/calculate.js

```diff
@@ -0,0 +1,133 @@
+import operate from './operate';
+
+function isNumber(item) {
+  return !!item.match(/[0-9]+/);
+}
+
```

Comment:
- [ ] Good job adding `operate.js` and `calculate.js` files as the previous reviewer requested, however, you are required to add them to a logic directory as shown in the requirements below, this would help in the proper organization of your code.

<img width="691" alt="Screenshot 2023-07-12 at 21 56 20" src="https://github.com/DjibrilM/math-magician/assets/64774936/3193c137-3cd5-4f34-b073-5d15bda1e5c4">